stdlib/unittest/mock.pyi

```diff
@@ -180,7 +180,9 @@ class _patch(Generic[_T]):
     @overload
     def __call__(self, func: Callable[..., _R]) -> Callable[..., _R]: ...
     def decorate_class(self, klass: _TT) -> _TT: ...
-    def decorate_callable(self, func: Callable[..., _R]) -> Callable[..., _R]: ...
+    def decorate_callable(self, func: _F) -> _F: ...
```

Comment:
The TypeVar doesn't seem right because the decorator adds extra arguments.

Reply:
One day I'll make a change to unittest.mock that doesn't have any mistakes

Reply:
unittest.mock as a whole is a mistake